content/docs/started/getting-started-gke.md

```diff
@@ -23,6 +23,10 @@ brings the following advantages:
   * [Cloud Identity-Aware Proxy (Cloud IAP)](https://cloud.google.com/iap/) 
     makes it easy to securely connect to Jupyter and other
     web apps running as part of Kubeflow.
+  * Basic auth service supports simple username/password access to your 
+    Kubeflow. It is an alternative to Cloud IAP service. We recommend IAP for 
```

Comment:
nit: make this sub-bullet points it will help with readability I think.

Reply:
done.